physics/cu_c3_driver.F90

```diff
@@ -135,7 +135,7 @@ subroutine cu_unified_driver_run(ntracer,garea,im,km,dt,flag_init,flag_restart,&
 !  ruc variable
    real(kind=kind_phys), dimension (:),   intent(in)  :: hfx2,qfx2,psuri,ca_deep
    real(kind=kind_phys), dimension (:,:), intent(out) :: ud_mf,dd_mf,dt_mf
-   real(kind=kind_phys), dimension (:),   intent(out) :: raincv,cld1d,rainevap
+   real(kind=kind_phys), dimension (:),   intent(out) :: raincv,cld1d,maxupmf,rainevap
```

Comment:
Please double-check that `maxupmf` is always given a value (i.e. that it is not being set within an `if` statement).

Reply:
@grantfirl Thanks for your suggestion. It will be updated as below when I update this PR.
            maxupmf(i)=0.
            if(forcing(i,6).gt.0.)then
              maxupmf(i)=maxval(xmb(i)*zu(i,kts:ktf)/forcing(i,6))
            endif